### PR TITLE
Fix bikeshed errors

### PR DESCRIPTION
**Pull request by npm1**
Fixes some bikeshed errors:
* Explicitly use w3c in Status
* Indentation error
* Change RFC to the latest version

***
<a href="https://pr-preview.s3.amazonaws.com/npm1/FedCM/pull/272.html" title="Last updated on Jun 8, 2022, 5:15 PM UTC (262ace8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/272/f6834f1...npm1:262ace8.html" title="Last updated on Jun 8, 2022, 5:15 PM UTC (262ace8)">Diff</a>

### CODE REVIEW

**Review by samuelgoto — APPROVED** [review]
LGTM

**Comment by samuelgoto** [comment]
Resolve the conflicts and I'll merge?

File: spec/index.bs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Title: Federated Credential Management API
 Shortname: FedCM
 Level: 1
-Status: CG-DRAFT
+Status: w3c/CG-DRAFT
```
Comment (cbiesinger) [inline spec/index.bs:5]: This should not be necessary if you update your bikeshed installation

Comment (cbiesinger) [reply spec/index.bs:5]: Yeah -- `pip install bikeshed --upgrade`

**Pull request merged**